lib/transform.js

```diff
@@ -0,0 +1,81 @@
+const vegadataflow = require('../vendor/vega-dataflow.js');
+
+
+var tx = vegadataflow.transforms,
+    changeset = vegadataflow.changeset;
+
+
+export function aggregate(fields, operations, data) {
```

Comment:
This is really my preference and also these two are not equivalent, please see for more info - https://stackoverflow.com/questions/336859/var-functionname-function-vs-function-functionname